Dockerfile

```diff
@@ -15,4 +21,9 @@ RUN \
  ls | grep -v app.py > to-be-removed.txt &&\
  for file in `cat to-be-removed.txt`; do rm $file; done
 
+COPY ./ ${THOTH_GRAPH_SYNC_DIR}
```

Comment:
This will need to stay before the RUN command, otherwise `pip3 install -r requiremnets.txt` will fail.